secrets/dataplane-observability-config.yaml.sample

```diff
@@ -0,0 +1,40 @@
+---
+# [required]
+remote_write_configuration:
+  # [required] V2 URL endpoint of the Prometheus instance to be used by the data planes
+  # to send metrics. The Prometheus instance has to offer a Remote Write endpoint.
+  # If oidc_configuration is set that endpoint has to be offered under the
+  # /api/v1/metrics/<tenant>/api/v1/receive" URL path, where <tenant> is defined by the
+  # --observability-red-hat-sso-tenant Fleet Manager flag.
```

Comment:
Is there a reason why we still define the tenant as a flag rather than within this config?

Reply:
Aside from what peter's was mentioning, the reason we didn't move it already in the config is because we kind of reached to the conclusion that the issue is that KFM shouldn't be worried about the tenant at all. From the point of view of KFM the only thing that is needed is to have knowledge of a Prometheus Remote Write Endpoint. Whether Observatorium (a specific implementation of a remote write endpoint) has the concept of a tenant it is irrelevant. From the point of view of KFM it's just an URL.
Therefore, the long term solution for that wouldn't be moving the tenant to the config, but removing the concept of tenant altogether

Reply:
makes sense now, thanks for the clarification!